Major rework of ecosystem pages (#3060)
open-telemetry/opentelemetry.io: content/en/ecosystem/integrations.md

spelling: cSpell:ignore containerd buildx quarkus
---

> A non-exhaustive list of libraries, services or apps which integrate

Reviewer: ```suggestion
A non-exhaustive list of libraries, services and apps that integrate
```

OpenTelemetry **natively** to emit traces, metrics and logs.

{{% ecosystem/integrations-table %}}

To have a library, service or app listed,
[add an entry to the list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/integrations.yaml)
and submit a PR.

You will need to include a link to the documentation that explains how
OpenTelemetry is used to enable observability.

Note, that this list is for libraries, services or apps which integrate
OpenTelemetry **natively**.

If you provide external integration of OpenTelemetry for any kind of library,
service or app, [add it to the registry](/ecosystem/registry/adding) <br /> If
you adopt OpenTelemetry for Observability as an end user and do not provide any
kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).
<br /> If you provide a solution that consumes OpenTelemetry to offer

> Observability to end users, see [Vendors](/ecosystem/vendors).

Reviewer: See the copyedits I made to the corresponding part of the Adopters page. Please make similar changes here.

spelling: cSpell:ignore containerd buildx quarkus
---

A non-exhaustive list of libraries, services and apps that integrate
OpenTelemetry **natively** to emit traces, metrics and logs.

{{% ecosystem/integrations-table %}}

> To have your library, service or app listed, submit a PR with an entry added to

Reviewer: ```suggestion
To have your library, service, or app listed, submit a PR with an entry added to
```

the
[integrations list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/integrations.yaml).
The entry must include the following:

> - a link to the main page of your library, service or app.
> - a link to the documentation that explains how OpenTelemetry is used to enable
>   observability.

Reviewer: ```suggestion
- Link to the main page of your library, service, or app
- Link to the documentation that explains how enable observability using OpenTelemetry
```

> Note, that this list is for libraries, services or apps which integrate

Reviewer: ```suggestion
Note that this list is for libraries, services, and apps that integrate
```

OpenTelemetry **natively**.

> If you provide external integration of OpenTelemetry for any kind of library,
> service or app, [add it to the registry](/ecosystem/registry/adding) <br /> If
> you adopt OpenTelemetry for Observability as an end user and do not provide any
> kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).
> <br /> If you provide a solution that consumes OpenTelemetry to offer
> Observability to end users, see [Vendors](/ecosystem/vendors).

Reviewer: ```suggestion
If you provide external integration of OpenTelemetry for any kind of library,
service, or app, then [add it to the registry](/ecosystem/registry/adding).

If you adopt OpenTelemetry for Observability as an end user and do not provide any
kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).

If you provide a solution that consumes OpenTelemetry to offer
Observability to end users, see [Vendors](/ecosystem/vendors).
```
